### PR TITLE
[test-codereviewhub] Test

### PR DESCRIPTION
**Pull request by Luismahou**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98040318%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98040445%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98040854%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98040997%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98041119%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98041123%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98042504%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98042668%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23discussion_r29489125%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23discussion_r29489250%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23discussion_r29489404%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23discussion_r29489414%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98043984%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98044341%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23discussion_r29489487%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23discussion_r29490057%22%2C%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23discussion_r29490069%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23issuecomment-98040318%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20is%20a%20comment%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A19%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/856327%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Luismahou%22%7D%7D%2C%20%7B%22body%22%3A%20%22Yeah%20alright.%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A21%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6703966%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/benletchford%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A30%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/511428%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/alexanderdickson%22%7D%7D%2C%20%7B%22body%22%3A%20%22Image%5Cr%5Cn%5Cr%5Cn%21%5Bimage%5D%28https%3A//cloud.githubusercontent.com/assets/40389/7426692/fa838030-f00e-11e4-9101-159596c3fad2.png%29%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A33%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/40389%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kongakong%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A35%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/40389%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kongakong%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A35%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/40389%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kongakong%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%3Agun%3A%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A39%3A37Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/856327%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Luismahou%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A40%3A07Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/856327%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Luismahou%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%3Agun%3A%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A42%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/40389%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kongakong%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A43%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/40389%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kongakong%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2067090765d014c9fc064875b9b9f100b54d3c9b5e%20test.txt%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/Zeetings/git-templates/pull/3%23discussion_r29489125%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20like%20your%20use%20of%20%27test%27%20in%20this%20sentence.%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A21%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6703966%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/benletchford%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5BNP%5D%20What%20if%20this%20isn%27t%20a%20test%2C%20but%20life%20itself%20is%20a%20test%3F%20Then%20this%20file%20need%20no%20longer%20exist%3F%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A28%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/511428%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/alexanderdickson%22%7D%7D%2C%20%7B%22body%22%3A%20%22This%20comment%20should%20work%20as%20an%20ACK%20for%20codereviewhub.%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A37%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/856327%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Luismahou%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A38%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/856327%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Luismahou%22%7D%7D%2C%20%7B%22body%22%3A%20%22So%20anyone%20can%20mark%20a%20task%20as%20completed...%22%2C%20%22created_at%22%3A%20%222015-05-01T04%3A43%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/40389%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kongakong%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-05-01T05%3A17%3A07Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/856327%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Luismahou%22%7D%7D%2C%20%7B%22body%22%3A%20%22Yep%2C%20and%20anyone%20can%20mark%20the%20task%20as%20pending.%22%2C%20%22created_at%22%3A%20%222015-05-01T05%3A17%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/856327%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Luismahou%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20test.txt%3AL1%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/Luismahou%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/856327%3Fv%3D3%22%7D%2C%20%22https%3A//github.com/alexanderdickson%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/511428%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/Luismahou'></a><a href='https://github.com/alexanderdickson'></a>

- [ ] <a href='#crh-comment-Pull 67090765d014c9fc064875b9b9f100b54d3c9b5e test.txt 1'></a> &nbsp;<b><a href='https://github.com/Zeetings/git-templates/pull/3#discussion_r29489125'>File: test.txt:L1</a></b>
- <a href='https://github.com/benletchford'></a> I like your use of 'test' in this sentence.
- <a href='https://github.com/alexanderdickson'></a> [NP] What if this isn't a test, but life itself is a test? Then this file need no longer exist?
- <a href='https://github.com/Luismahou'></a> This comment should work as an ACK for codereviewhub.
- <a href='https://github.com/Luismahou'></a> :+1:
- <a href='https://github.com/kongakong'></a> So anyone can mark a task as completed...
- <a href='https://github.com/Luismahou'></a> :+1:
- <a href='https://github.com/Luismahou'></a> Yep, and anyone can mark the task as pending.
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/Zeetings/git-templates/pull/3#issuecomment-98040318'>General Comment</a></b>
- <a href='https://github.com/Luismahou'></a> This is a comment
- <a href='https://github.com/benletchford'></a> Yeah alright.
- <a href='https://github.com/alexanderdickson'></a> :shipit:
- <a href='https://github.com/kongakong'></a> Image
image
- <a href='https://github.com/kongakong'></a> :shipit:
- <a href='https://github.com/kongakong'></a> :shipit:
- <a href='https://github.com/Luismahou'></a> :shipit::gun:
- <a href='https://github.com/Luismahou'></a> :shipit:
- <a href='https://github.com/kongakong'></a> :shipit::gun:

<a href='https://www.codereviewhub.com/Zeetings/git-templates/pull/3?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/Zeetings/git-templates/pull/3?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/Zeetings/git-templates/pull/3?approve=0'></a>&nbsp;<a href='https://github.com/Zeetings/git-templates/pull/3'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by Luismahou** [comment]
This is a comment

**Comment by benletchford** [comment]
Yeah alright.

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a test
```
Comment (benletchford) [inline test.txt]: I like your use of 'test' in this sentence.

Comment (alexanderdickson) [inline test.txt]: hi

**Comment by alexanderdickson** [comment]
:shipit:

**Comment by kongakong** [comment]
Image

image

**Comment by kongakong** [comment]
:shipit:

**Comment by kongakong** [comment]
:shipit:

File: test.txt (hunk 1 above)
Comment (Luismahou) [inline test.txt]: This comment should work as an ACK for codereviewhub.

Comment (Luismahou) [inline test.txt]: :+1:

**Comment by Luismahou** [comment]
:shipit::gun:

**Comment by Luismahou** [comment]
:shipit:

**Comment by kongakong** [comment]
:shipit::gun:

**Comment by kongakong** [comment]
:+1:

File: test.txt (hunk 1 above)
Comment (kongakong) [inline test.txt]: So anyone can mark a task as completed...

Comment (Luismahou) [inline test.txt]: :+1:

Comment (Luismahou) [inline test.txt]: Yep, and anyone can mark the task as pending.

**Pull request closed without merging**